src/map/modifier.h

```diff
@@ -544,6 +544,7 @@ enum class Mod
     CARBUNCLE_LVL_BONUS       = 1041, // Carbuncle: Lv.+ (Increases Carbuncle's base level above 99)
     CAIT_SITH_LVL_BONUS       = 1042, // Cait Sith: Lv.+ (Increases Cait Sith's base level above 99)
     ENHANCES_MANA_CEDE        = 74,   // Bonus % to Mana Cede effect, +1 = 1%
+    SUMMONING_MAGIC_CAST      = 1181, // Summoning magic casting time reduction in seconds
```

Comment:
There's probably a comment towards the end of this file that needs updating, saying you've used this number - otherwise seems fine 👍